entry/src/entry.rs

```diff
@@ -1434,4 +1434,19 @@ mod tests {
             info!("{} {}", time, res);
         }
     }
+
+    #[test]
+    fn test_hash_transactions() {
```

Comment:
basically just a test to make sure we're not doing something silly like re-ordering signatures in this function that wraps around calls to `MerkleTree`.

Reply:
cc/ @2501babe added this because I had a minor panic this morning when looking at code and realizing this `hash_transactions` didn't do exactly what I thought it did.

This test just ensures that the orde of transactions in an entry is captured in the hash, which is something that we require for SIMD83 to work properly.